example/tk/jsonnetfile.json

```diff
@@ -19,11 +37,29 @@
       },
       "version": "master"
     },
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/grafana/jsonnet-libs.git",
+          "subdir": "prometheus"
+        }
+      },
+      "version": "master"
+    },
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/grafana/tempo.git",
+          "subdir": "operations/tempo-mixin"
+        }
+      },
+      "version": "main"
```

Comment:
I was thinking about that a bit.  I'm happy to make the change, though I was also thinking that a symlink to the development location would be sufficient for testing and developing.  It would just mean that you wouldn't want to run `jb` while developing since then it would remove the symlink, which is a little more overhead.  I'll make the change to ease development.

Reply:
How would the symlink work? If this is easy to setup and documented that's also fine for me.